tests/test_machine_restore.py

```diff
@@ -0,0 +1,146 @@
+# Copyright 2015 rpaas authors. All rights reserved.
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+
+import datetime
+import time
+import unittest
+import redis
+from freezegun import freeze_time
+from mock import patch, call
+from rpaas import storage, tasks
+from rpaas import scheduler
+from hm import managers, log
+from hm.model.host import Host
+
+tasks.app.conf.CELERY_ALWAYS_EAGER = True
+
+
+class RestoreHostError(Exception):
+    pass
+
+
+class FakeManager(managers.BaseManager):
+
+    host_id = 0
+    hosts = []
+    fail_ids = []
```

Comment:
you are using these as instance attributes. you should move to the `__init__` method. Something like:

```
def __init__(self, config=None):
    self.host_id = 0
```